CMakeLists.txt

```diff
@@ -240,12 +240,6 @@ if(OpenCL_FOUND)
   endif()
 endif()
 
-find_package(rocalution)
-if(ROCALUTION_FOUND)
-  set(HAVE_ROCALUTION 1)
-  set(COMPILE_BDA_BRIDGE 1)
-endif()
-
```

Comment:
Mmh, can't find the place where you search rocalution now. Seems like we might not do that anymore. Is that intentional?

Reply:
Should be lines 291-294.
We also removed the explicit `find_package(rocsparse)`, it should be done automatically because they are listed in `opm-simulators-prereqs.cmake`.

Reply:
Ah, thanks.